src/stories/Library/Modals/modal-find-on-shelf/modal-find-on-shelf.scss

```diff
@@ -0,0 +1,25 @@
+.modal-find-on-shelf {
+  padding: 0;
+  overflow-y: scroll;
+  align-items: flex-start;
+
+  @include breakpoint-m {
+    padding: 0 215px $s-4xl 215px;
```

Comment:
We actually have a variable for 215px: `$s-7xl`.

Reply:
This piece of code doesn't exist anymore :) 
But good to know we have, oh, so many variables for sizes 😛